src/test/java/tw/edu/ntu/lowerbound10hours/jlaskhibernate/DatabaseOperationTest.java

```diff
@@ -0,0 +1,34 @@
+package tw.edu.ntu.lowerbound10hours.jlaskhibernate;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+import org.hibernate.Criteria;
+import org.testng.annotations.Test;
+import tw.edu.ntu.lowerbound10hours.jlask.Jlask;
+
+public class DatabaseOperationTest {
+
+  private Hibernate db = new Hibernate();
+
+  public DatabaseOperationTest() {
+    db.drop_all();
+    db.init_app(new Jlask());
+  }
+
+  @Test
+  public void testInsert() {
+    Post post = new Post("Test Post1", "It's a testing");
+    db.getSession().save(post);
+    db.getTransaction().commit();
+  }
+
+  @Test
+  public void testSearch() {
+    Criteria criteria = db.getSession().createCriteria(Post.class);
```

Comment:
Does this test have to be after the above one?
I think there should be a @Before and @After, kind of like https://dzone.com/articles/testing-databases-junit-and